src/main/java/teammates/ui/webapi/request/InstructorCreateRequest.java

```diff
@@ -0,0 +1,62 @@
+package teammates.ui.webapi.request;
+
+import javax.annotation.Nullable;
+
+/**
+ * The create request for an instructor to be created.
+ */
+public class InstructorCreateRequest extends BasicRequest {
+    @Nullable
+    private String id;
+
+    private String name;
+    private String email;
+    private String roleName;
+
+    @Nullable
+    private String displayName;
+
+    @Nullable
+    private Boolean isDisplayedToStudent;
+
+    public InstructorCreateRequest(String id, String name, String email, String roleName, String displayName,
+                                   Boolean isDisplayedToStudent) {
+        this.id = id;
+        this.name = name;
+        this.email = email;
+        this.roleName = roleName;
+        this.displayName = displayName;
+        this.isDisplayedToStudent = isDisplayedToStudent;
```

Comment:
Default `isDisplayedToStudent` to `true` if not present. This field is mandatory and thus not nullable. Try to use an `Optional`.

By the time this request DTO reaches the action, the `isDisplayedToStudent` must be non-null. See what you need to do in the declaration of the member variable, in the constructor and in the `validate` method.

Reply:
Should I enforce this check when calling the API endpoint? I can add a check to the `validate` method.